internal/parser/translate.go

```diff
@@ -739,3 +799,27 @@ func PortDefFromIntStr(is intstr.IntOrString) kongstate.PortDef {
 	}
 	return kongstate.PortDef{Mode: kongstate.PortModeByNumber, Number: is.IntVal}
 }
+
+// -----------------------------------------------------------------------------
+// Utilities - Gateway APIs
+// -----------------------------------------------------------------------------
+
+// convertGatewayMatchHeadersToKongRouteMatchHeaders takes an input list of Gateway APIs HTTPHeaderMatch
+// and converts these header matching rules to the format expected by go-kong.
+func convertGatewayMatchHeadersToKongRouteMatchHeaders(headers []gatewayv1alpha2.HTTPHeaderMatch) (map[string][]string, error) {
```

Comment:
nit: `fromKongRouteMatchHeaders` would be just as descriptive in the call site, and brief.

Reply:
It seems the "from" would be incorrect here, because it's "to" kong route match headers :thinking:

I played around with a couple different variations to try and shorten in up, but I kept wanting more.

I'll keep it in mind while I continue to work on this area of the code.